source/NVDAObjects/window/edit.py

```diff
@@ -684,16 +684,17 @@ def getTextWithFields(self, formatConfig: Optional[Dict] = None) -> textInfos.Te
 			formatConfig=config.conf["documentFormatting"]
 		textRange=self._rangeObj.duplicate
 		textRange.collapse(True)
-		if not formatConfig["detectFormatAfterCursor"]:

```

Comment:
Is this exposed in the GUI? Can it be removed if no longer observed?

Reply:
From another conversation with @michaelDCurran, if this is the only usage, we can remove it from the settings dialog.

Reply:
There is another usage at: `source/textInfos/offsets.py:570`
Snippet:
```python
	def getTextWithFields(self, formatConfig: Optional[Dict] = None) -> textInfos.TextInfo.TextWithFieldsT:
		if not formatConfig:
			formatConfig=config.conf["documentFormatting"]
		if self.detectFormattingAfterCursorMaybeSlow and not formatConfig['detectFormatAfterCursor']:
			field,(boundStart,boundEnd)=self._getFormatFieldAndOffsets(self._startOffset,formatConfig,calculateOffsets=False)
			text=self.text
			return [textInfos.FieldCommand('formatChange',field),text]
		commandList=[]
		offset=self._startOffset
		while offset<self._endOffset:
```